pyi.py

```diff
@@ -1085,6 +1085,7 @@ def _check_for_type_aliases(
             self.error(node, Y026.format(suggestion=unparse(new_node)))
 
     def visit_Name(self, node: ast.Name) -> None:
+        self.generic_visit(node)
```

Comment:
This method was flagged by flake8-bugbear's B906. Calling `generic_visit()` here probably isn't, strictly speaking, necessary for us here, but it also doesn't do any harm. Bugbear is flagging this here because the `ctx` attribute of an `ast.Name` node is itself an AST node that can be visited by an `ast.NodeVisitor` subclass.